libethashseal/genesis/ropsten.cpp

```diff
@@ -25,7 +25,8 @@ R"E(
 		"homesteadForkBlock": "0x00",
 		"EIP150ForkBlock": "0x00",
 		"EIP158ForkBlock": "0x0a",
-		"byzantiumForkBlock": "0xfffffffffffffff",
+		"daoHardforkBlock": "0x00",
```

Comment:
it's 0 by default, probably has no effect, since dao fork happens only in transition